src/Commands/RemoveResizesCommand.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Rapidez\ImageResizer\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\File;
+
+class RemoveResizesCommand extends Command
+{
+    protected $signature = 'rapidez:resizes:delete {store? : Store ID from Magento}';
+
+    protected $description = 'Delete the resized images.';
+
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    public function handle()
+    {
+        $storeModel = config('rapidez.models.store');
+        $stores = $this->argument('store') ? $this->argument('store') : $storeModel::all()->pluck('id');
+
+        if (is_a($stores, 'Illuminate\Support\Collection')) {
```

Comment:
I suggest doing
```suggestion
        if ($stores instanceof \Illuminate\Support\Collection) {
```
or
```suggestion
        if (is_iterable($stores)) {
```

as `instanceof` is a language construct and thus faster, and because it's argument is the class itself instead of a string editors will be friendlier with it.

It iterable is more flexible in that it will accept an array or anything that works in a foreach loop as well

Reply:
👌  i think a is_iterable wouldn't work as we can loop through an object with a foreach too so i think an single object would be iterable too.